src/lib/Ska/Starcheck/Dark_Cal_Checker.pm

```diff
@@ -142,12 +158,40 @@ sub maneuver_parse{
 					+ timestring_to_secs($manvr->{DURATION}),
 					duration => timestring_to_mins($manvr->{DURATION}),
 					);
-		
+                # mark each maneuver to a replica
+		if (defined $is_replica{$manvr->{oflsid}}){
```

Comment:
Since "old style" dark cals have DC_T? and DFC_? values for oflsid instead of obsids ("new style" basically has dot oflsid == obsid), old style manvr entries for replicas weren't getting a True to get in this if block.